staging/src/k8s.io/kubelet/pkg/apis/stats/v1alpha1/types.go

```diff
@@ -227,6 +227,9 @@ type MemoryStats struct {
 	// hugepages).
 	// +optional
 	RSSBytes *uint64 `json:"rssBytes,omitempty"`
+	// The amount of swap currently used.
+	// +optional
```

Comment:
Can you add that this is currently alpha and include the feature gate it's controlled by?

Reply:
ok. I will add it.